models/articles.model.js

```diff
@@ -1,15 +1,35 @@
 const db = require("../db/connection");
 
-function fetchArticleById(id) {
+function fetchArticleById(article_id) {
   return db
-    .query(`SELECT * FROM articles WHERE articles.article_id = $1;`, [id])
+    .query(`SELECT * FROM articles WHERE articles.article_id = $1;`, [
+      article_id,
+    ])
     .then(({ rows }) => {
       article = rows[0];
       if (!article) {
-        return Promise.reject({ status: 404, msg: "article does not exist" });
+        return Promise.reject({ status: 404, msg: "Article does not exist" });
       }
       return article;
     });
 }
 
-module.exports = fetchArticleById;
+function fetchArticles() {
+  return db
+    .query(
+      `SELECT articles.*, COUNT(comments.article_id) AS comment_count 
```

Comment:
make sure the comment count for your articles is accurate with what's in your test data - counting instance of article id might not be the best thing to count if we want the number of comments (see if there's a difference is if you count comment id for example)